packages/client/src/schema/files.ts

```diff
@@ -165,7 +165,8 @@ export class XataFile {
   public transform(...options: ImageTransformations[]) {
     return {
       url: transformImage(this.url, ...options),
-      signedUrl: transformImage(this.signedUrl, ...options)
+      signedUrl: transformImage(this.signedUrl, ...options),
+      metadataUrl: transformImage(this.url, ...options, { format: 'json' })
```

Comment:
Not sure if it's worth adding a `metadataSignedUrl`, normally `signedUrl` is used for displaying it in the DOM not for programmatic access.